relate/utils.py

```diff
@@ -163,6 +165,21 @@ def __call__(self, request):
 # }}}
 
 
+def get_site_name():
+    # type: () -> Text
+    from django.conf import settings
+    return getattr(settings, "RELATE_SITE_NAME", "RELATE")
+
+
+def render_email_template(template_name, context=None, request=None, using=None):
+    # type: (Text, Optional[Dict], Optional[HttpRequest], Optional[bool]) -> Text
+    if context is None:
+        context = {}
+    context.update({"RELATE": _(get_site_name())})
```

Comment:
I kind of feel that `{{ site_name }}` or `{{ relate_site_name }}` would be more self-explanatory.

Reply:
(Plus, the upper-case variable name is a bit weird.)

Reply:
I agree. Though my current code is to reduce the work of changing the i18n literals, I'll change that.

Reply:
Will change to `relate_site_name`.